charts/nginx/values.yaml

```
@@ -74,3 +74,6 @@ ports:
   defaultBackendHTTP: 8080
   healthz: 10254
   metrics: 10254
+
+# Election ID to use for status update
+electionID: ingress-controller-leader
```

AFAIK camelCase has `election_id` as `electionId` <https://capitalizemytitle.com/camel-case/>
```suggestion
electionId: ingress-controller-leader
```